spec/latest/core/index.html

```diff
@@ -555,9 +555,12 @@ <h3>Documenting a Web API</h3>
 
     <p>Keep in mind that operations specified in an <i>ApiDocumentation</i>
       may fail at runtime as either resources or the <i>ApiDocumentation</i>
-      itself have changed since they have been retrieved. A simple strategy
-      to try to recover from such an error is to reload the
-      <i>ApiDocumentation</i>.</p>
+      itself have changed since they have been retrieved. Also operation details
+      like <i>hydra:returns</i> or <i>hydra:possibleStatus</i> may vary at runtime,
+      thus these interpreted at runtime. A simple strategy to try to recover from
+      such a situation is to reload the <i>ApiDocumentation</i> and redo all
+      operations that were based on the <i>ApiDocumentation</i> (or at least those
+      that lead to the current failure).</p>
```

Comment:
I don't see how this is relevant. Furthermore, I think this is incorrect and misleading.

Why would the client redo operations which succeeded prior to reloading the API Documentation?

>  operations that were based on the ApiDocumentation

The language here is also weird. How is an operation based on the ApiDocumentation? This terminology is not used anywhere else AFAICT.

Reply:
I've reworded _operation_ with _pre-computations_. In general, I can imagine a form is generated from a given ApiDocumentation, but the received resource does not match the ApiDocumentation. Resolution would require the application to reload ApiDocumentation, regenerate the form and apply the received resource again.

I've also added an example involving error message shown to the user with option to return to e.g. home screen.